src/tests/runner.test.js

```diff
@@ -0,0 +1,304 @@
+import * as assert from 'assert';
```

Comment:
Plenty more cases I want us to unit test here -- but enough to get deployed for manual testing